### ISSUE TITLE
20.10 stuck at black screen on soft reboot

### ISSUE THREAD

**Issue by dhitchcock** [pop-os/pop#1335]
**Distribution (run `cat /etc/os-release`):**
 
Pop!_OS 20.10
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

not sure

**Issue/Bug Description:**

I have an original Galago UltraPro (galu1) and I upgraded from 20.04 -> 20.10.  Once 20.10 was installed, if I tell the computer to reboot, the current session shuts down because you see the bios screen, then the screen goes black and it won't start up again.  I have to hold the power button down to shut down.  Once it's off, I can turn it back on and it starts up just fine. 

I thought maybe it was just something funky from the upgrade, so I did a clean install of 20.10, same issue.  Just to make sure I wasn't crazy and it wasn't hardware or something, I did a clean install of 20.04.  Gone, works fine again.

**Steps to reproduce (if you know):**

Install Pop!_OS 20.10 on a galu1 if you still have any of those around.  I just did a clean install, boot it up, open a terminal and run 'sudo reboot' and the subsequent boot hangs.

Also, it hangs on the first boot after install is complete too.

**Comment by dhitchcock** [comment]
Oh nooooo...  After an apt upgrade 20.04 does this too now, it was the kernel upgrade to 5.8

**Comment by akari-dcoakley** [comment]
@dhitchcock - Exact same issue here. On a Galago UltraPro too. Also happened in a similar way to me. I updated Ubuntu 20.04 and started having issues with reboot/shutdown. Then I upgraded to 20.10 in the hope the issue would disappear. However, the issue remains.

**Comment by DylanC** [comment]
@dhitchcock - Exact same issue here. On a Galago UltraPro too. Also happened in a similar way to me. I updated Ubuntu 20.04 and started having issues with reboot/shutdown. Then I upgraded to 20.10 in the hope the issue would disappear. However, the issue remains.

**Comment by dhitchcock** [comment]
Yeah it does happen in plain Ubuntu  20.10 also, so there could be an upstream bug.  Interestingly it does NOT happen in Fedora 33, which is also a 5.8 kernel.  So that points to some ubuntu patches perhaps and not the 5.8 kernel itself.   ¯\_(ツ)_/¯

**Comment by dhitchcock** [comment]
I bet it's this kernel bug: https://bugzilla.kernel.org/show_bug.cgi?id=208737

Fedora doesn't suffer it though, weird.  I tried the latest Manjaro too and that was a disaster.

**Comment by DylanC** [comment]
@dhitchcock - Had a quick read through. Yeah that sounds like it! It's really quite annoying since I use my PopOS machine for work everyday. I have to endure this bug daily. :disappointed:

**Comment by dhitchcock** [comment]
To escape this I switched to just Ubuntu 20.04, but just recently it pushed out kernel 5.8.0-36-generic so I was fearful that my plain Ubuntu was going to start doing the same thing.... but nope, it's fine.  So I actually re-wiped my system to see if it was fixed in Pop then, but alas.  Still broke.

Ubuntu kernel 5.8.0-36 in 20.04 - fine.
Pop kernel 5.8.0-7630 in 20.10 - bad.

So pull in them ubuntu updates I guess!  I can tell every engineer at System76 is losing a lot of sleep over this one I want them to finally get some rest. :^)
